src/script/components/panel/PanelActions/PanelActions.tsx

```diff
@@ -37,18 +37,19 @@ export interface PanelActionsProps {
 const PanelActions: React.FC<PanelActionsProps> = ({items}) => {
   return (
     <ul css={listCSS}>
-      {items.map(({click, identifier, icon, label}) => (
-        <li key={identifier}>
-          <button className="panel__action-item" onClick={click} data-uie-name={identifier} type="button">
-            <span className="panel__action-item__icon">
-              <Icon name={icon} />
-            </span>
-            <span data-uie-name={`${identifier}-item-text`} className="panel__action-item__text">
-              {label}
-            </span>
-          </button>
-        </li>
-      ))}
+      {items &&
+        items.map(({click, identifier, icon, label}) => (
```

Comment:
Maybe we could make the items not optional. Does it really make sense to have them this way?!

Reply:
I agree and eventually render the component conditionally. There is no need to render it when there are no items.